test/threads.jl

```diff
@@ -200,7 +202,19 @@ close(proc.in)
             done[] = true
             close(timer)
         end
-        @test success(proc)
+        if Sys.iswindows()
+            if !success(proc)
+                @error """
+                Failed test: "spawn and wait *a lot* of tasks in @profile" (n = $n)
+
+                If you notice this error message, please share the test output in
+                https://github.com/JuliaLang/julia/issues/43124.  The error and
+                backtrace should be printed above.
+                """ proc.exitcode proc.termsignal
```

Comment:
Not sure if this is a good approach. But I think we want to gather info about this.